### ISSUE TITLE
items USA 1940 Census in webtrees is not the same is in official census

### ISSUE THREAD

**Issue by Joppla** [fisharebest/webtrees#1307]
please make the items the same.
items for the official 1940 Census are described in https://1940census.archives.gov/questions-asked/ or 

    1. Street, Avenue, Road, etc
    2. House Number

Household Data

    3. Number of household in order of visitation
    4. Home owned (O) or rented (R)
    5. Value of home or monthly rental if rented
    6. Farm? (Yes or No)

Name

    7. Name of each person whose usual place of residence on April 1, 1940, was in this household.

Relation

    8. Relationship of this person to the head of the household

Personal Description

    9. Sex
    10. Color or Race
    11. Age at Last Birthday
    12. Marital Status
    13. Attended school or college at any time since March 1, 1940?
    14. Highest grade of school completed

Place of Birth

    15. Place of Birth [Note: supposed to list the state or foreign country]

Citizenship

    16. Citizenship of the foreign born

Residence, April 1, 1935

    17. City, town or village having 2,600 or more inhabitants. If less, enter "R"
    18. County
    19. State (or Territory or foreign country)
    20. On a farm? (Y or N)

For Persons 14 Years Old and Over – Employment Status:

    21. Was this person at work for pay or profit in private or nonemergency Govt. work during the week of March 24-30? (Y or N)
    22. If not, was he at work on, or assigned to, public Emergency Work (WPA, NYA, CCC, etc.) during the week of March 24-30? (Y or N)
    If neither:
    23. Was this person seeking work? (Y or N)
    24. If not seeking work, did he have a job, business, etc.? (Y or N)

For Persons Answering "No" to questions 21-24:

    25. Indicate whether engaged in home housework (H), unable to work (U), or other (O).

If "Yes" in Column 21:

    26. Number of hours worked during week of March 24-30, 1940

If "Yes" in Column 22 or 23:

    27. Duration of unemployment up to March 30, 1940 – in weeks

Occupation, Industry, and Class of Worker:

    28. Occupation
    29. Industry
    30. Class of worker
    31. Number of weeks worked in 1939 (equivalent full-time weeks)

Income in 1939 (12 months ending Dec. 31, 1939):

    32. Amount of money, wages, or salary received
    33. Did this person receive income of $50 or more from sources other than money wages or salary (Y or N)
    34. Number of Farm Schedule

There are also items 35-50 but they where asked to only 5% of the population.

you can see them on the attached pdf.
[1940-census-schedule.pdf](https://github.com/fisharebest/webtrees/files/1218967/1940-census-schedule.pdf)

NB The 1930 Census has other items.

**Comment by LarsEighner** [comment]
I'm not much of a hacker, but this seems to work for me. No doubt the phrasing could be improved. Change file extension. 
[CensusOfUnitedStates1940.txt](https://github.com/fisharebest/webtrees/files/2952753/CensusOfUnitedStates1940.txt)

**Comment by fisharebest** [comment]
Thanks.  The corresponding test script will need updating too, if you have time.

https://github.com/fisharebest/webtrees/blob/master/tests/app/Census/CensusOfUnitedStates1940Test.php

**Comment by LarsEighner** [comment]
Sorry. Cannot understand OOP. Tried for 20 years to learn it.

**Comment by Ziris85** [comment]
@fisharebest, here you go. I hope this works/helps! This is how I ended up framing the 1940 census adjustments on mine. I've included patch files for both of the relevant 1940 census files below:
[1940.zip](https://github.com/fisharebest/webtrees/files/4637725/1940.zip)

While I was at it, since we're now within a couple of years of the 1950 census becoming public record, I figured now would be a good time to write up the entry for that, included below:
[1950.zip](https://github.com/fisharebest/webtrees/files/4637729/1950.zip)

Notable here is the exclusion of the Supplementary questions that a lot of these censuses asked, though these were missing before. However, I'd like to offer a suggestion on how to implement them, but I'll open a new issue for that since that exceeds the scope of this issue.
